### DISCUSSION TITLE
Divert calls to different system

### DISCUSSION THREAD

**Discussion by cpg178** [chuot/rdio-scanner#262] [Q&A]
For a few weeks now I have been having an issue where certain TGs of a trunked system want to go to another system ID, since that system ID does not exist it rejects them. Here is an example newcall: system=4 talkgroup=1005 file=20220915_100921Luzerne__North_Simulcast_TRAFFIC__TO_1005_FROM_187090.mp3 no matching system/talkgroup

I could turn on auto populate at the highest level but then I would have 2 different systems within rdio for the same thing. This is being sent from SDRTrunk with an unnamed alias so every TG is just passed along as its TGID 
![image](https://user-images.githubusercontent.com/4503742/190469148-0137615e-f676-4c32-b203-1c5d2e3f4535.png)

**Comment by chuot** [comment]
On the `Playlist Editor`, make sure under the `Channels` tab that the `System` label matches the System Label on the Rdio Scanner's side. If it doesn't then Rdio Scanner will try the create a new System with a new sequential SystemID.
